styles.css

```diff
@@ -1,4 +1,7 @@
 /* Friends of Girl Develop It Columbus styles */
+:root { /                              /* new */
```

Comment:
There's an errant "/" after the opening curly-brace, please remove that.

Reply:
k that's fixed